vendor/engines/split_accounts/spec/services/order_detail_splitter_spec.rb

```diff
@@ -13,8 +12,10 @@
     end
   end
 
+  let(:order) { build_stubbed(:order) }
+
   let(:order_detail) do
-    build_stubbed(:order_detail).tap do |order_detail|
+    build_stubbed(:order_detail, order: order).tap do |order_detail|
```

Comment:
Not saying it needs to be changed now, but is there a reason we're not adding the things inside `tap` just using the standard `build_stubbed` options hash?